packages/sidenav/src/helpers/PageWrapper/PageWrapper.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { PUSH, FULLSCREEN, OVERLAY } from "../../constants/variants";
+import StyledPageWrapper, { Overlay } from "./PageWrapper.styled";
+
+const PageWrapper = React.memo(({ overlayProps, ...props }) => (
+  <>
+    {props.variant === OVERLAY && props.expanded && (
+      <Overlay data-testid="overlay-pageWrapper" {...overlayProps} />
+    )}
+    <StyledPageWrapper data-testid="pageWrapper-navbar" {...props} />
+  </>
+));
+
+PageWrapper.defaultProps = {
+  expanded: false,
+  offsetTop: "0",
+  collapsedWidth: "56px",
+  expandedWidth: "270px",
+  variant: PUSH,
+};
+
+PageWrapper.propTypes = {
+  children: PropTypes.node,
+  /**
+   Sidebar state value (collapsed/extended)
+  */
+  expanded: PropTypes.bool,
```

Comment:
We need at least one story for this component, so this documentation shows up in storybook.